components/forms/FindMultisigForm.js

```diff
@@ -18,26 +17,25 @@ class FindMultisigForm extends React.Component {
     };
   }
 
-  handleChange = (e) => {
+  handleChange(e) {
     this.setState({
       [e.target.name]: e.target.value,
     });
-  };
+  }
 
-  handleSearch = async () => {
```

Comment:
The re-formating of the class methods broke the app. In order for this method to work, we would need to add `this.handleSearch = this.handleSearch.bind(this)` in the constructor on every class. I prefer the arrow function method as it feels a bit drier to me. Is there an eslint rule that disagrees with the arrow function format?

Reply:
Good point. I also realized this when testing yesterday.

The issue is described very well [here](https://reactjs.org/docs/handling-events.html), especially starting at this paragraph:

> If calling bind annoys you, there are two ways you can get around this. If you are using the experimental public class fields syntax, you can use class fields to correctly bind callbacks:

Turns out the syntax that was used before is correct but experimental. The eslint parser cannot parse it:

```
  27:25  error  Parsing error: Unexpected token =
```

So I went with the second option, which is pretty straight forward.

I think long term it would be good to turn all React components to either functions or classes for consistency.

Reply:
I agree, I'd like to eventually convert everything to use functions and hooks. So i'm ok with using the `.bind(this)` to fix this for now.

Reply:
Just saw the commit you added, I'm ok with this method too :)